src/ansi.rs

```diff
@@ -0,0 +1,1933 @@
+//! ANSI Terminal Stream Parsing.
```

Comment:
The `SPDX` header should be used with `APACHE` license, because the `alacritty_terminal` is APACHE only.

Reply:
Fixed in https://github.com/alacritty/vte/pull/85/commits/b2d518d87d94512bcae22d7bd3a8c07b945bf7a2.